pom.xml

```diff
@@ -1,17 +1,80 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 
-        <modelVersion>4.0.0</modelVersion>
+    <modelVersion>4.0.0</modelVersion>
 
-        <!-- project coordinates -->
-        <groupId>dreamteam.battleship</groupId>
-        <artifactId>battleship</artifactId>
-        <version>1.0</version>
-        <packaging>pom</packaging>
+    <!-- project coordinates -->
+    <groupId>dreamteam.battleship</groupId>
+    <artifactId>battleship</artifactId>
+    <version>1.0</version>
+    <packaging>pom</packaging>
+
+    <modules>
+        <module>battleship-logic</module>
+        <module>battleship-service</module>
+    </modules>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.5.1</version>
+                <configuration>
+                    <source>1.8</source>
+                    <target>1.8</target>
```

Comment:
Do you plan on EVER having source and target apart? If not, use one property for both.